public/docs/_examples/dependency-injection/dart/lib/car/car_no_di.dart

```diff
@@ -13,10 +13,10 @@ class Car {
     engine = new Engine();
     tires = new Tires();
   }
-
   //#enddocregion car-ctor
 
   // Method using the engine and tires
-  drive() => '$description car with ${engine.cylinders} cylinders and ${tires.make} tires.';
+  drive() => '$description car with '
```

Comment:
I'll check, but I believe we're just following the original TypeScript example here.

We follow Dart conventions where possible (e.g. using types for APIs), while trying to keep the code similar (e.g. names).